### PR TITLE
fix: correcting the readme file

### PR DESCRIPTION
**Pull request by MostlyWhat**
The readme file used to say Deno but I feel like using "CDN Demo" seems more fitting.

### CODE REVIEW

**Comment by ST-DDT** [comment]
Deno is a thing too: https://deno.land/

File: README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ const randomEmail = faker.internet.email(); // Kassandra.Haley@erich.biz
 const randomCard = faker.helpers.createCard(); // random contact card containing many properties
 ```
 
-### Deno
+### CDN Demo
```
Comment (ST-DDT) [inline README.md:62]:
```suggestion
### CDN/Deno
```

**Review by ST-DDT — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MostlyWhat) [reply README.md:62]: Thats good

Comment (Shinigami92) [inline README.md:62]: I assume something was mixed up while merging main :thinking: 
This line should address `Deno`

**Review by Shinigami92 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MostlyWhat) [reply README.md:62]: I thought I merged the change above

**Comment by MostlyWhat** [comment]
I think I modified it.

**Review by MostlyWhat — COMMENTED** [review]
Review Please

**Review by Shinigami92 — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Pull request merged**